context/default/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM python:3.10-bookworm
```

Comment:
Could it make sense to go for an alpine based image to get a smaller container out of this? Also, maybe go with a Python 3.12 image, since the 3.10 have quite a few vulnerabilities?

https://hub.docker.com/layers/library/python/3.10-bookworm/images/sha256-68763c64e640977ad219a74dde8c445f6fac5d2a229c4e01058a88fcdab2249c

Reply:
This was the first take at adapting the original setup into a more decoupled environment.

Adapting the configuration to `alpine` seems aspireable, yet out of scope for this cycle.

Let's see how well it runs with Python 3.12, thanks for the (security) hint.

```suggestion
ARG PYTHON_TAG=3.12-bookworm
```

Reply:
I've added a commit that implements this change after prior testing. Pretalx runs well with Python 3.12.

703579f